components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -35,6 +47,12 @@
  */
 @interface MDCDialogPresentationController : UIPresentationController
 
+/**
+ An object conforming to MDCDialogPresentationControllerDelegate.
```

Comment:
Please provide a description of this property's intended use rather than a description of it's declaration.

For example, see UIPresentationController's delegate docs: https://developer.apple.com/documentation/uikit/uipresentationcontroller/1618329-delegate